css/media.css

```diff
@@ -0,0 +1,500 @@
+/*@media only screen and (min-width: 768px) {
+	.product-tile {
+		width: 50%;
+	}
+}*/
+
+/*@media only screen and (min-width: 1000px) {
+	.product-tile {
+		width: 25%;
+	}
+}*/
+
+/*@media only screen and (max-width: 1170px) {
+
+}*/
+
+@media only screen and (max-width: 1024px) {
+	.header-content ,
+	.poster-content ,
+	.product-content {
+		max-width: 1024px;
+	}
+
+	.header-container ,
+	.poster-container ,
+	.product-container ,
+	.product-items {
+		max-width: 930px;
+	}
+
+	.poster-container{
+		height: 252px;
+	}
+		
+	.product-tile {
+		max-width: 208px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_Axion/pullRequest?prid=5457135)